doc/faq.rst

```diff
@@ -26,6 +26,20 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
+--------------------------------------------------------------
+For bug reports or feature requests, please make use of the
+`issue tracker on Github <https://github.com/scikit-learn/scikit-learn/issues>`_
+or the `mailing list <https://mail.python.org/mailman/listinfo/scikit-learn>`_.
+
+Helping users is important to us, but we suggest searching 
```

Comment:
I would rather phrase it as "The quickest way to get replies to your questions is posting on SO or Cross-validated with the scikit-learn or machine learning tags. These sites may already have an answer for your question and provide a convenient search interface. You can also post questions on the mailing list, though that often has longer turn-around times. Please don't ask general machine learning questions on the mailing list.

Please don't email the authors..."

If you like, you can also mention gitter. But we can also do that in another PR.